src/helm/benchmark/metrics/classification_metrics.py

```diff
@@ -46,14 +47,14 @@ def evaluate_instances(self, request_states: List[RequestState]) -> List[Stat]:
             num_correct = 0
             for reference in request_state.instance.references:
                 if reference.is_correct:
+                    if reference.output.text == "No Label":
```

Comment:
No label should just be indicated by no output references with a correct label.

Also need to change upstream scenarios to not output "No Label" as a reference.